app/static/styles/main.css

```diff
@@ -107,19 +114,19 @@ body.resume header div div a.resume, body.contact header div div a.contact {
 }
 
 .profile h1 {
-        font-size: 50px;
-        font-weight: 700;
-        color: white;
+    font-size: 50px;
+    font-weight: 700;
+    color: white;
 }
 
 .profile img {
-        border-radius: 50%;
-        max-width: 100%;
-        max-height: 100%;
-        margin-left: auto;
-        margin-right: auto;
-        display: block;        
-        padding: 20px;
+    border-radius: 50%;
+    max-width: 100%;
+    max-height: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+    padding: 20px;
 }
 
```

Comment:
idrk where to put it but since "MLH Fellow" heading was made a link it has a line at the bottom which looks kinda weird. also it might be clearer to users that the heading is a link if we give it a hover (maybe like the ones for the navigation bar?)

Reply:
I removed the hyperlink underline but the hyperlink works. Maybe you can help me add the hover effect? I'm not too good at those

Reply:
sure I can do that when I edit the links